remote_syslog.go

```diff
@@ -53,6 +54,11 @@ func tailOne(file string, excludePatterns []*regexp.Regexp, logger *syslog.Logge
 // Tails files speficied in the globs and re-evaluates the globs
 // at the specified interval
 func tailFiles(globs []string, excludedFiles []*regexp.Regexp, excludePatterns []*regexp.Regexp, interval RefreshInterval, logger *syslog.Logger, severity syslog.Priority, facility syslog.Priority, poll bool) {
+	if len(globs) == 0 {
```

Comment:
I actually propose merging config and command-line arguments early in the
process. I can't think of anything that requires keeping both.
On Mon, Jun 29, 2015 at 2:45 PM Leon Sodhi <notifications@github.com> wrote:

> In remote_syslog.go
> <https://github.com/papertrail/remote_syslog2/pull/73#discussion_r33459382>
> :
>
> > @@ -53,6 +54,11 @@ func tailOne(file string, excludePatterns []*regexp.Regexp, logger *syslog.Logge
> >  // Tails files speficied in the globs and re-evaluates the globs
> >  // at the specified interval
> >  func tailFiles(globs []string, excludedFiles []*regexp.Regexp, excludePatterns []*regexp.Regexp, interval RefreshInterval, logger *syslog.Logger, severity syslog.Priority, facility syslog.Priority, poll bool) {
> > +	if len(globs) == 0 {
>
> I've just answered my own question: yes and no :-) If we check for files
> earlier on via Files(), that should cover files from both the config &
> CLI, and will allow us to display an error even when damonized. That should
> make this redundant and allow it to be removed.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/papertrail/remote_syslog2/pull/73/files#r33459382>.
>